ApollosProject/apollos-prototype
# Add TestUtils to be used by all package test suites

## DESCRIPTION

### What does this PR do, or why is it needed?
For the sake of time, when creating the new API packages I duplicated a lot of testing code, mostly test helpers and mocks, into each new package. Now that the dust is starting settle on the package structure I moved the `testHelper` into the `server-core package, and I moved all mocks back into a one place and turned it into a module. 

### What design trade-offs/decisions were made?
n/a

### How do I test this PR?
Run the tests - ensure that no snapshots change and that all tests are still passing. 

### What automated tests did you write?
n/a - all existing tests still pass.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
